### ISSUE TITLE
Prism Launcher Lags When Using Keyboard Focus and/or Crashes Screen Readers

### ISSUE THREAD

**Issue by blindgoofball** [PrismLauncher/PrismLauncher#6074]
### Operating System

Windows

### Version of Prism Launcher

Prism Launcher 1.1.0

### Version of Qt

unable to access due to the bug

### Description of bug

When launching prism launcher with a screen reader active (tested with the free NVDA screen reader), the window refuses to accept keyboard focus when using alt+tab. Additionally, when using alternative methods to interact with the window such as clicking with the mouse, my screen reader crashed or lagged multiple times. Specifically, I opened the settings window, and my screen reader completely crashed, and when using mouse clicks to change the user interface, the screen reader lagged for several seconds before reading anything, and keyboard navigation was completely unusable, making the launcher completely inaccessible to screen readers when QT usually has decent accessibility.

### Steps to reproduce

* Open prism launcher with NVDA running
* Try to use tab to navigate (this will not work)
* Use alt+tab to focus a window other than prism launcher, then use the same keystroke to refocus it (the window will not properly receive keyboard focus)
* Click on the settings button from the toolbar (this will likely lag or crash the screen reader)
* Change something in settings (the screen reader will not speak anything for several seconds)
* Try keyboard navigation with tab and arrow keys in settings (this will also fail)

### This issue is unique

- [x] I have searched the issue tracker and did not find an issue describing my bug.
